content/en/docs/reference/kubectl/jsonpath.md

```diff
@@ -65,18 +70,19 @@ Given the JSON input:
-`.` or `[]`         | child operator            | `{.kind}`, `{['kind']}` or `{['name\.type']}`                   | `List`
-`..`                | recursive descent         | `{..name}`                                                      | `127.0.0.1 127.0.0.2 myself e2e`
-`*`                 | wildcard. Get all objects | `{.items[*].metadata.name}`                                     | `[127.0.0.1 127.0.0.2]`
-`[start:end:step]` | subscript operator        | `{.users[0].name}`                                              | `myself`
-`[,]`               | union operator            | `{.items[*]['metadata.name', 'status.capacity']}`               | `127.0.0.1 127.0.0.2 map[cpu:4] map[cpu:8]`
-`?()`               | filter                    | `{.users[?(@.name=="e2e")].user.password}`                      | `secret`
-`range`, `end`      | iterate list              | `{range .items[*]}[{.metadata.name}, {.status.capacity}] {end}` | `[127.0.0.1, map[cpu:4]] [127.0.0.2, map[cpu:8]]`
-`''`                | quote interpreted string  | `{range .items[*]}{.metadata.name}{'\t'}{end}`                  | `127.0.0.1      127.0.0.2`
+Function            | Description                  | Example                                                         | Result
+--------------------|------------------------------|-----------------------------------------------------------------|------------------
+`text`              | the plain text               | `kind is {.kind}`                                               | `kind is List`
+`@`                 | the current object           | `{@}`                                                           | the same as input
+`.` or `[]`         | child operator               | `{.kind}`, `{['kind']}` or `{['name\.type']}`                   | `List`
+`..`                | recursive descent            | `{..name}`                                                      | `127.0.0.1 127.0.0.2 myself e2e`
+`*`                 | wildcard. Get all objects    | `{.items[*].metadata.name}`                                     | `[127.0.0.1 127.0.0.2]`
+`[start:end:step]` | subscript operator           | `{.users[0].name}`                                              | `myself`
+`[,]`               | union operator               | `{.items[*]['metadata.name', 'status.capacity']}`               | `127.0.0.1 127.0.0.2 map[cpu:4] map[cpu:8]`
+`?()`               | filter                       | `{.users[?(@.name=="e2e")].user.password}`                      | `secret`
+`range`, `end`      | iterate list                 | `{range .items[*]}[{.metadata.name}, {.status.capacity}] {end}` | `[127.0.0.1, map[cpu:4]] [127.0.0.2, map[cpu:8]]`
+`''`                | quote interpreted string     | `{range .items[*]}{.metadata.name}{'\t'}{end}`                  | `127.0.0.1      127.0.0.2`
+`\`                 | escape termination character | `{.items[0].metadata.labels.kubernetes\.io/hostname}`           | `127.0.0.1`
```

Comment:
Why do you say it only escapes the period character?

Reply:
It's not only the period, it's the termination char, named from the jsonpath parser: https://github.com/kubernetes/kubernetes/blob/551790729f1d26d75c1d3fa1411e341eb367f9f3/staging/src/k8s.io/client-go/util/jsonpath/parser.go#L449-L459

So we can escape them by `\`, I just give an example of escaping `.`